installers/python-ci/radiasoft-download.sh

```diff
@@ -17,6 +17,9 @@ python_ci_main() {
             i=radiasoft/python3
             p+=( pykern )
             ;;
+        rslearn)
+            i=radiasoft/python3
```

Comment:
I think this can be added to the case above (ex `rsaccounting|rsconf|rslearn`) because rslearn will need pykern installed to work.

Reply:
thanks for catching that. 5e8464c58272fad2dae3202f50a7e1e9e9d05a28